.github/workflows/update-registry.yml

```diff
@@ -0,0 +1,82 @@
+name: Update Registry
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Safe version with deployment type'
+        required: true
+        type: choice
+        options:
+          - v1.3.0-canonical
+          - v1.3.0-eip155
+          - v1.4.1
+          - v1.5.0
+      chain_id:
+        description: 'Chain ID to add (e.g., 12345)'
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-registry:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
```

Comment:
Unrelated to this PR (so nothing to change here), but just wanted to make a mental note that the LTS version of Node is now 24, so we should probably change the CI to that at some point.

For now, leaving it at 22 to be consistent with the other jobs is preferred IMO.